sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/AppConfigurationReplicaClientFactory.java

```diff
@@ -25,7 +25,7 @@ public class AppConfigurationReplicaClientFactory {
      * @param clientBuilder builder for app configuration replica clients
      * @param configStores configuration info for config stores
      */
-    public AppConfigurationReplicaClientFactory(AppConfigurationReplicaClientsBuilder clientBuilder,
+    AppConfigurationReplicaClientFactory(AppConfigurationReplicaClientsBuilder clientBuilder,
```

Comment:
Looks like a little bit weird the factory depends on a builder here?

Reply:
Yeah, I think it's a naming issue, one creates the App Config Store stuff where each store has multiple instances of individual clients, which the other makes.